site_root/input/ImplementationGuide-KidsFirst.json

```diff
@@ -1,273 +1,289 @@
 {
```

Comment:
Most of the changes below except the new lines happen due to indent=4. Could you re-indent this file and the rest of the new resources with indent=2?

Reply:
@liberaliscomputing 
The change has to be made in the IG generation code: #131

Reply:
done!